src/gpl/src/nesterovBase.cpp

```diff
@@ -1899,6 +1983,24 @@ getOverlapArea(const Bin* bin, const Instance* inst) {
   }
 }
 
+
+// A function that does 2D integration to the density function of a
+// bivariate normal distribution with 0 correlation.
+// Essentially, the function being integrated is the product
+// of 2 1D probability density functions (for x and y). The means and standard deviation of the 
+// probablity density functions are parametarized.
+// In this function, I am using the closed-form solution of the integration.
+// The limits of integration are lx->ux and ly->uy
+static float calculateBiVariateNormalCDF(biNormalParameters  i)
```

Comment:
Is there a reference for this formula?  It's nice to provide one for non-obvious math.

Reply:
I added the equation being integrated as a comment.